### PR TITLE
Added negative roll effect

### PR DESCRIPTION
**Pull request by abhishek-iiit**
# as Part of SWOC

### CODE REVIEW

File: negative.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#construct the argument parse and parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--image", required=True, help="path to input image")
+args = vars(ap.parse_args())
+
+# reading the image
+img = cv2.imread((args["image"]))
+negative = 255 - img
+
+cv2.imwrite("assets/negative.jpg", negative)
```
Comment (smaranjitghose) [inline negative.py:16]: Can you please give an input and output image in the description of the PR as a sample

File: negative.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Import the dependencies
+import argparse
+import cv2 #image processing lib
+import numpy as np #matrix manipulation
+from time import sleep #for slowing down the process to make progress visible 
+from tqdm import tqdm as tqdm # for progess bar
```
Comment (smaranjitghose) [inline negative.py]: I guess we don't need the progress bar here as there is not much computationally expensive operations being done

File: negative.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+from tqdm import tqdm as tqdm # for progess bar 
+
+#construct the argument parse and parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--image", required=True, help="path to input image")
+args = vars(ap.parse_args())
+
+# reading the image
+img = cv2.imread((args["image"]))
+negative = 255 - img
```
Comment (smaranjitghose) [inline negative.py:14]: Try it for pink color(any shade you like) and let me know what we get.

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: negative.py (hunk 2 above)
Comment (abhishek-iiit) [reply negative.py]: Ok,will change..

File: negative.py (hunk 3 above)
Comment (abhishek-iiit) [reply negative.py:14]: Ok

File: negative.py (hunk 1 above)
Comment (abhishek-iiit) [reply negative.py:16]: Oh! Sorry.
 Yes, for sure will do this change

**Pull request merged**